pkg/server/server.go

```diff
@@ -182,3 +183,16 @@ func (host *yamlLanguageHost) InstallDependencies(req *pulumirpc.InstallDependen
 	// No dependencies to install for YAML
 	return nil
 }
+
+// GetProgramDependencies returns the set of dependencies required by the program.
+func (host *yamlLanguageHost) GetProgramDependencies(ctx context.Context, req *pulumirpc.GetProgramDependenciesRequest) (*pulumirpc.GetProgramDependenciesResponse, error) {
+	return &pulumirpc.GetProgramDependenciesResponse{}, nil
+}
+
+// About returns information about the runtime for this language.
+func (host *yamlLanguageHost) About(ctx context.Context, req *emptypb.Empty) (*pulumirpc.AboutResponse, error) {
+	return &pulumirpc.AboutResponse{
+		Executable: "pulumi-language-yaml",
+		Version:    version.Version,
+	}, nil
+}
```

Comment:
@Frassle this is what you'd expect, yeah?

Reply:
Executable and version should be empty for yaml (you can just not set them, or set them to "")

Reply:
Blank? That's surprising to me, is this not conveyed back to the user?

Reply:
This is intended to report information about the executable we're using for the runtime itself, not information about the language plugin. So e.g. dotnet reports back something like "/usr/bin/dotnet", "6.0". 
yaml doesn't have an executable for the runtime, so it should just be blank. The logic in about handles the case that these can come back as `""`.